app/utils/zip-agenda-files.js

```diff
@@ -39,7 +40,8 @@ async function fetchArchivingJob(agenda, mandateeIds, decisions= false) {
 }
 
 async function fetchArchivingJobForAgenda(agenda, mandateeIds, decisions, store) {
-  const job = await fetchArchivingJob(agenda, mandateeIds, decisions);
+  const extensions = ["pdf"]; // TODO get this value from control provided by design
```

Comment:
Does this TODO make this PR unmergable? Not sure of the design status.
We could default to pdf in this PR and improve frontend in a different PR to keep this moving.

Reply:
Design is available. Tasking @StuyckRuben with it today :-)